lib/fastlane/plugin/bundletool/actions/bundletool_action.rb

```diff
@@ -158,30 +158,37 @@ def self.puts_error!(message)
       def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :ks_path,
+                                       env_name: 'FL_BUNDLETOOL_KEYSTORE_FILE',
```

Comment:
@txuslee I just changed `FL_BUNDLETOOL_STORE_FILE` to `FL_BUNDLETOOL_KEYSTORE_FILE`

Let me know if its ok